R/pkg/R/DataFrame.R

```diff
@@ -78,6 +78,55 @@ dataFrame <- function(sdf, isCached = FALSE) {
 
 ############################ SparkDataFrame Methods ##############################################
 
+#' storageName
+#'
+#' Return a SparkDataFrame's name.
+#'
+#' @param x The SparkDataFrame whose name is returned.
+#' @family SparkDataFrame functions
+#' @rdname storageName
+#' @examples
+#'\dontrun{
+#' sparkR.session()
+#' path <- "path/to/file.json"
+#' df <- read.json(path)
+#' storageName(df)
+#'}
+#' @aliases storageName,SparkDataFrame-method
+#' @export
+#' @note storageName since 2.2.0
+setMethod("storageName",
+          signature(x = "SparkDataFrame"),
+          function(x) {
+            callJMethod(x@sdf, "name")
+          })
+
+#' storageName
+#'
+#' Set a SparkDataFrame's name. This will be displayed on the Storage tab in the UI if cached.
+#'
+#' @param x The SparkDataFrame whose name is to be set.
+#' @param name The SparkDataFrame name to be set.
+#' @family SparkDataFrame functions
+#' @return the SparkDataFrame renamed.
+#' @rdname storageName
+#' @examples
+#'\dontrun{
+#' sparkR.session()
+#' path <- "path/to/file.json"
+#' df <- read.json(path)
+#' storageName(df) <- "foo"
```

Comment:
since it won't be useful unless it is cached, suggest add in the example here `cache(df)`